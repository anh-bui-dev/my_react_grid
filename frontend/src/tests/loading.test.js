import { shallow, mount } from 'enzyme';
import React, { Component } from 'react';
import Loading from '../components/loading';

describe('Loading', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});