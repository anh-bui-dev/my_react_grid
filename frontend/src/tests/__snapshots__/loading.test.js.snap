// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Should render correctly 1`] = `
<div
  className="modal-mask"
>
  <div
    className="loading d-flex justify-content-center"
  >
    <div
      className="spinner-grow text-secondary"
      role="status"
    >
      <span
        className="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
